content/en/synthetics/browser_tests/test_results.md

```diff
@@ -47,90 +73,78 @@ Browser
 : The type of browser your test was executed from.
 
 Time ran
-: The time your test ran.
+: The amount of time that has passed since your test ran.
 
 Run type
-: The type of your test run (CI, fast retry, manually triggered, or scheduled).
-
-### Screenshots
-
-Browser tests contain screenshots for every executed test step. Screenshots help you visualize the journey your browser test went through.
+: The type of test run (CI, fast retry, manually triggered, or scheduled).
 
-### Page performance
-
-Every step in which a full URL loads contains page performance information.
+### RUM sessions
 
-#### User experience
+To view related sessions and available replays in the RUM Explorer, click **View Session in RUM**. To access a user session for a particular action or step in Session Replay, click **Replay Session**. For more information, see [Explore RUM & Session Replay in Synthetics][16].
 
-[Google's Core Web Vitals][1] are a set of three metrics designed to monitor a site's user experience. These metrics focus on giving you a view of load performance, interactivity, and visual stability. Each metric comes with guidance on the range of values that translate to a good user experience.
+### Screenshots and actions
 
-Synthetic Monitoring includes two available lab metrics: [Largest Contentful Paint][2] and [Cumulative Layout Shift][3].
+Every executed test step contains a screenshot of the step action, a link to the session in Session Replay, the step description, the starting URL for a given step, the step ID, the step duration, and page performance information.
 
-{{< img src="real_user_monitoring/browser/core-web-vitals.png" alt="Core Web Vitals summary visualization" style="width=80%" >}}
-
-[First Input Delay][4] is available if you are using [Real User Monitoring][5] to collect real user and field data. For more information, see [Monitoring Page Performance][6].
+### Page performance
 
-### Errors
+Synthetic Monitoring displays two [Core Web Vital metrics][6] ([Largest Contentful Paint][2] and [Cumulative Layout Shift][3]) as pills to the right of each step URL. [First Input Delay][4] is available if you are using [Real User Monitoring][5] to collect real user data. For more information, see [Monitoring Page Performance][6].
 
-The **Errors** panel shows the error, its type (`js`/`network`), and status (network status code).
+### Errors and warnings
 
-The type of error is logged during interaction with the page. It corresponds to the errors collected between the time the page is opened and the time interaction with that page is realized.
+Click the **Errors** pill to access the **Errors & Warnings** tab and examine a list of errors separated by error type (`js` or `network`) and status (the network status code).
```

Comment:
Would suggest adding a screenshot that makes it clear what the "Error pill" looks like, as that isn't a UI/UX term I'm familiar with.